simulations/simulation.py

```diff
@@ -1,31 +1,25 @@
 import asyncio
+from asyncio import sleep
 import logging
 import os
-import random
 import shutil
 import time
-from asyncio import sleep
-from typing import Optional
-
-import yappi
-
-from bami.dkg.community import DKGCommunity
-from ipv8.messaging.interfaces.statistics_endpoint import StatisticsEndpoint
-
-from bami.basalt.community import BasaltCommunity
-from bami.skipgraph.community import SkipGraphCommunity
+from typing import Dict, Optional, Type
 
 from ipv8.configuration import ConfigBuilder
+from ipv8.messaging.interfaces.statistics_endpoint import StatisticsEndpoint
 from ipv8.taskmanager import TaskManager
+from ipv8.types import Community
 from ipv8_service import IPv8
+import yappi
 
-from simulation.discrete_loop import DiscreteLoop
-from simulation.simulation_endpoint import SimulationEndpoint
-
+from common.discrete_loop import DiscreteLoop
+from common.network import SimulatedNetwork
+from common.simulation_endpoint import SimulationEndpoint
 from simulations.settings import SimulationSettings
 
 
-class BamiSimulation(TaskManager):
+class BamiSimulation:
```

Comment:
I'm not sure why we had this class as a subclass of `TaskManager`, might have been something from the past.